### ISSUE TITLE
Frontend design

### ISSUE THREAD

**Issue by hl** [uberous/lista#8]
Nicos can you take a stab at this?

**Comment by panoply** [comment]
Should I begin building thing, based off example design we cooked up last night? I will add SASS variables for color changes, but general "wireframe" I can begin on? @hl

**Comment by hl** [comment]
Yes @panoply you can start whenever, my main focus is to launch it, design can always be adjusted later if needed.
